kernelci/api/helper.py

```diff
@@ -272,17 +272,25 @@ def should_create_node(self, rules, node, parent=None):
     def create_job_node(self, job_config, input_node,
                         runtime=None, platform=None):
         """Create a new job node based on input and configuration"""
+        jobfilter = input_node.get('jobfilter')
         job_node = {
             'kind': job_config.kind,
             'parent': input_node['id'],
             'name': job_config.name,
             'path': input_node['path'] + [job_config.name],
             'group': job_config.name,
             'artifacts': {},
+            'jobfilter': jobfilter,
             'data': {
                 'kernel_revision': input_node['data']['kernel_revision'],
             },
         }
+        # if jobfilter not null, verify if job_config.name exist in jobfilter
+        if jobfilter:
+            if job_config.name not in jobfilter:
```

Comment:
```suggestion
        if jobfilter and job_config.name not in jobfilter:
```

Minor nit, optional

Reply:
No, i appreciate for such nits, i should not be lazy/low attention on such issues and produce more accurate code.
So, thanks for spotting, updated